src/query/service/src/interpreters/interpreter_select.rs

```diff
@@ -125,13 +127,29 @@ impl SelectInterpreter {
             }
         }
 
-        build_query_pipeline(
+        let mut build_res = build_query_pipeline(
             &self.ctx,
             &self.bind_context.columns,
             &physical_plan,
             self.ignore_result,
         )
-        .await
+        .await?;
+
+        // consume stream
+        if let Some(req) = build_update_multi_stream_meta_req(&self.ctx, &self.metadata).await? {
```

Comment:
It's better to add a `info!` log for this consume, to make it easy to track the action of a action.

Reply:
> It's better to add a `info!` log for this consume, to make it easy to track the action of a action.

Thanks, added.